### PR TITLE
add runtime parameters to disable some subch2 rates

### PR DESCRIPTION
**Pull request by zingale**

### CODE REVIEW

**Comment by zingale** [comment]
This relies on  https://github.com/pynucastro/pynucastro/pull/290

File: networks/subch2/subch2.py [hunk 1]
```diff
@@ -55,7 +55,12 @@ def doit():
             node_size=500, node_font_size=11, node_color="#337dff", node_shape="s",
             Z_range=(1,29))
 
-    net = StarKillerCxxNetwork(libraries=[subch], symmetric_screening=True)
+    # these are the rates that we are going to allow to be optionally
+    # zeroed
+    r1 = subch.get_rate("p_c12__n13")
+    r2 = subch.get_rate("he4_n13__p_o16")
+
+    net = StarKillerCxxNetwork(libraries=[subch], symmetric_screening=True, rate_params=[r1, r2])
```
Comment (maxpkatz) [inline networks/subch2/subch2.py:63]: I think this is out of date now

Comment (zingale) [reply networks/subch2/subch2.py:63]: ah right

Comment (zingale) [reply networks/subch2/subch2.py]: updated

**Review by maxpkatz — APPROVED** [review]

**Pull request merged**